src/api/getConversation.ts

```diff
@@ -0,0 +1,22 @@
+import { idbDatabase, CONVERSATION_STORE_NAME } from './idb';
+import type { ConversationHistory } from '../types';
+
+export async function getConversation(
+  id: string,
+): Promise<Omit<ConversationHistory, 'conversation'> | null> {
```

Comment:
I like the idea of storing a "preview" of the conversation history without all of the content. Can we establish this as a named type so its purpose is clear when its reused?